src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -153,6 +163,22 @@ private void setupListView() {
         });
     }
 
+    private void setupIssueSelectedTrigger(){
+        // filter right mouse button out. Choose only the left mouse button for triggering IssueSelectedEvent
+        listView.addEventFilter(MouseEvent.MOUSE_PRESSED, e -> {
+            if (e.isSecondaryButtonDown()) {
+                shouldTriggerIssueSelectedEvent = false;
```

Comment:
A better way to do this is to modify setOnItemSelected to pick up which key is being pressed, then pass that to the callback above via the parameter.